lib/app/views/onc_program.erb

```diff
@@ -648,6 +648,27 @@
               value: instance.bulk_url,
               })%> 
 
+
+        <div class="form-group"
+          data-requiredby="<%= test_set.variable_required_by(:attachment_requires_token).map {|seq| seq.sequence_name}.join(',')%>"
+          data-prerequisite="attachment_requires_token"
+          id="op_attachment_requires_token"
+          >
+          <label style="display:block">Attachment URL Authorization
+            <small>Select SMART Bearer token if attachments are served from a location that accepts the same bearer token used for the SIngle Patient API.  
+            Select "Self-signed URL" if the attachments encode authorization information within the URL and bearer tokens are not required.  
```

Comment:
It seems inconsistent that `Self-signed URL` is quoted, but `SMART Bearer Token` isn't.